src/api/app/views/webui2/webui/project/_subprojects_table.html.haml

```diff
@@ -0,0 +1,11 @@
+%h3 #{table_title} projects of #{project}
+%table.responsive.table.table-striped.table-bordered.subproject-table
+  %thead
+    %tr
+      %th #{table_title} Project
```

Comment:
It now says `Parent Project`, `Subproject Project` and `Sibling Project`. I approved the PR, but I think it could be better if it's `Parent Project`, `Subproject` and `Sibling Project`.

Reply:
Why singular?

Reply:
https://ux.stackexchange.com/questions/48417/singular-or-plural-label-for-html-table-or-list

Reply:
We are just passing it via the table_title.. please check it out @dmarcoux